datacenter/ucp/2.2/guides/access-control/deploy-stack-to-collection.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Deploy stack resources to a collection 
+description: Learn how to manage user access to stack resources by using collections. 
+keywords: UCP, authentication, user management, stack, collection, role
+---
+
+A stack file is the definition of an application, and in the stack file,
+you can apply access labels to specify access to application resources.
+UCP interprets an access label to be the name of a collection that users are
+granted access to by an administrator.
+[Learn about managing access to resources by using collections](manage-access-with-collections.md).  
+
+Specify a collection for a resource by adding a `labels:` dictionary to the
+stack file and assigning the `com.docker.ucp.access.label` key. For example,
+a stack file that puts a resource in the `database` collection might look
+similar to this:  
+
+```none
+services:
+   db:
+     image: mysql:5.7
+     deploy:
+      labels:
+        com.docker.ucp.access.label: database
+```
+
+If you don't specify access labels in the stack file, resources are placed in
+the user's default collection when the stack deploys.
+
+You can place a stack's resources into multiple collections, but most of the
+time, you won't need to do this.
+
+## Deploy a stack file resource to a collection
+
+In this example, the following legacy stack file deploys a MySQL service with
+the access label `database`. For this deployment to succeed, an administrator
+must create a collection at `/Shared/Legacy/database` and
+[grant user permissions](grant-permissions.md) to access it.
+
+```none
```

Comment:
It would be nice to have a compose file with 2 services but without a lot of complexity.
Try this one out (you'll need to add the labels)
```
version: '3.1'

services:

  wordpress:
    image: wordpress
    ports:
      - 8080:80
    environment:
      WORDPRESS_DB_PASSWORD: example

  mysql:
    image: mysql:5.7
    environment:
      MYSQL_ROOT_PASSWORD: example
```